README.md

```diff
@@ -69,7 +69,7 @@ The binaries built by this repository can be run with any VMM which
 can act as a vhost-user frontend. Typically they have been tested with
 [QEMU](https://www.qemu.org) although the rust-vmm project does
 provide a [vhost-user
-frontend](https://github.com/rust-vmm/vhost/tree/main/src/vhost_user)
+frontend](https://github.com/rust-vmm/vhost/blob/main/crates/vhost/src/vhost_user/frontend.rs)
```

Comment:
I think it'd be much better if it linked to the directory that also displays the `README.md` instead of the source code file: https://github.com/rust-vmm/vhost/tree/main/crates/vhost